### ISSUE TITLE
Support Request

### ISSUE THREAD

**Issue by TheFalseProfit** [atomiclabs/ledger-kmd-reward-claim#7]
Hello AtomicLabs,

Google Chrome is up to date
Version 73.0.3683.75 (Official Build) (64-bit)

Ledger Nano S Firmware - Secure Element 1.4.2
Ledger Komodo App - 1.2.10
(Both say Up to date on Ledger Live)

I get the following error when attempting to use the ledger Komodo Claim Form at:
https://atomiclabs.github.io/ledger-kmd-reward-claim/
(this occurs in Chrome, Firefox and Opera).

Error
Ledger device: Invalid parameter received (0x6b00)

Check Rewards Button works, and gives the following information:

_________________start
Account 1
125.18286611 KMD
+ 2.46334987 KMD Claimable Rewards
UTXOs
Address	Value	Locktime	Rewards	Rewards Stop Accruing
Address	Value	Locktime	Rewards	Rewards Stop Accruing
RUmGiSq2WF62W4iHfMT9ysT1BD7hmxHjSL	24.998 KMD	
0.10565697 KMD	2 months ago
RTNmhndGn9h6u2KZJKniqypDMLcdAmWeKM	18.998 KMD	
0.0802458 KMD	4 months ago
RWyFTR7o8fbhAoYqJdhsvQuCGWm9zvwWBv	3.198 KMD	
0 KMD	in 2 months
RJFbZKgMcbg631nWtWMRPmVLL8E7dG1SEc	20.998 KMD	
0.85696166 KMD	in 2 months
RH1i9SeYU2jSXaNYw3AqmfwEyQmwfZRAbk	31.096 KMD	
1.55034595 KMD	2 months ago
RRd26BX9W2Lrv1g2ZqeRn4KX2q1ST16CkS	25.89486611 KMD	
0 KMD	N/A
Breakdown
2.59321038 KMD	Rewards accrued
0.12966051 KMD	5% service fee
0.0002 KMD	Network transaction fee
2.46334987 KMD	Total claimable amount
________________end

I then hit the Claim Rewards Button resulting in:

__________________start
Claiming Rewards

You should receive a total of 127.64621598 KMD to the new unused address: RSbGKhhcprBQTdtNYAFrckf3FzfKwZqCe9
There will also be a 5% service fee of 0.12966051 KMD to: RQzv9tVzCHXN9j3kF5qNzms555zzLC9HQF

Connect and unlock your Ledger, then open the Komodo app on your device.
Confirm the address on your device matches the new unused address shown above.
Approve the transaction on your device after carefully checking the values and addresses match those shown above.
Broadcasting the reward claim transaction to the network.
_______________end

This results in EITHER a single quick address that matches RSbGKhhcprBQTdtNYAFrckf3FzfKwZqCe9 and then a U2F TIMEOUT (shown below 1)
or I get two quick 'confirm address' events on the ledger with the second address never appearing on the ledger resulting in error 2 (shown below 2)

1.
Claiming Rewards

You should receive a total of 127.64621598 KMD to the new unused address: RSbGKhhcprBQTdtNYAFrckf3FzfKwZqCe9
There will also be a 5% service fee of 0.12966051 KMD to: RQzv9tVzCHXN9j3kF5qNzms555zzLC9HQF

Connect and unlock your Ledger, then open the Komodo app on your device.
Confirm the address on your device matches the new unused address shown above.
Approve the transaction on your device after carefully checking the values and addresses match those shown above.
Broadcasting the reward claim transaction to the network.
Error
Failed to sign with Ledger device: U2F TIMEOUT

2.
Claiming Rewards

You should receive a total of 127.64624811 KMD to the new unused address: RSbGKhhcprBQTdtNYAFrckf3FzfKwZqCe9
There will also be a 5% service fee of 0.12966221 KMD to: RQzv9tVzCHXN9j3kF5qNzms555zzLC9HQF

Connect and unlock your Ledger, then open the Komodo app on your device.
Confirm the address on your device matches the new unused address shown above.
Approve the transaction on your device after carefully checking the values and addresses match those shown above.
Broadcasting the reward claim transaction to the network.
Error
Ledger device: Invalid parameter received (0x6b00)

**Comment by lukechilds** [comment]
```
Ledger Nano S Firmware - Secure Element 1.4.2
Ledger Komodo App - 1.2.10
(Both say Up to date on Ledger Live)
```

Those versions are very old, the changes required to make this app work weren't released until version 1.3.0 of the Komodo app, https://github.com/LedgerHQ/ledger-app-btc/pull/84

There have also been many improvements since then.

**Comment by TheFalseProfit** [comment]
Thanks. Solved.
Issue arose from using Win 7.
Win 10 and latest Ledger Live Manager showed different Updates.
